ts/store/actions/wallet/cards.ts

```diff
@@ -44,13 +44,15 @@ export const cardsFetched = (
 });
 
 export const selectCardForDetails = (
-  card: CreditCard | number
+  card: CreditCard | CreditCardId
```

Comment:
this looks weird, and I'm not sure it makes sense semantically, can we just use `CreditCardId` ?

Reply:
Yeah there isn't a good reason for this, this function initially accepted a CreditCard, then I needed to pass this function an id as well, so I used this ^ for backward compatibility. But it definitely makes sense to only use CreditCardId